### PR TITLE
toggle switch widget - docs

### PR DESCRIPTION
**Pull request by arpitnath**
**Fixes** #540 

**Documentation for Widget Reference: Toggle Switch Widget**

**Screenshot**
Screenshot 2021-08-18 at 2 23 11 PM

### CODE REVIEW

**Comment by arpitnath** [comment]
need to change the alt attribute of the images

File: docs/docs/widgets/toggle-switch.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+The Toggle Switch Widget should be used if we want to make a binary choice, 
+such as turning something **on or off** or **enable or disable**.
+
+<img class="screenshot-full" src="/img/widgets/toggle-switch/toggle-switch.gif" alt="ToolJet - Toggle Switch Widget" height="420"/>
+
+
+## Event: onChange
+This event is triggered whenever toggle switch button is clicked, and hence the
+states are changed.
```
Comment (Navaneeth-pk) [inline docs/docs/widgets/toggle-switch.md:17]: `, and hence the states are changed.` not required.

File: docs/docs/widgets/toggle-switch.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Property : Label
+This property specifies a label that identifies the control to the user.
```
Comment (Navaneeth-pk) [inline docs/docs/widgets/toggle-switch.md:20]: `This property can be used to set a label for the switch.`

File: docs/docs/widgets/toggle-switch.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+Toggle Switch allows the user to change a setting between two states.
+
+The Toggle Switch Widget should be used if we want to make a binary choice, 
+such as turning something **on or off** or **enable or disable**.
+
+<img class="screenshot-full" src="/img/widgets/toggle-switch/toggle-switch.gif" alt="ToolJet - Toggle Switch Widget" height="420"/>
+
+
+## Event: onChange
+This event is triggered whenever toggle switch button is clicked, and hence the
```
Comment (Navaneeth-pk) [inline docs/docs/widgets/toggle-switch.md:16]: `whenever the toggle switch is`

File: docs/docs/widgets/toggle-switch.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
+sidebar_position: 2
+---
+
+# Toggle Switch
+
+Toggle Switch allows the user to change a setting between two states.
+
+The Toggle Switch Widget should be used if we want to make a binary choice,
```
Comment (Navaneeth-pk) [inline docs/docs/widgets/toggle-switch.md:9]: `The toggle switch widget`

File: docs/docs/widgets/toggle-switch.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+---
+sidebar_position: 2
+---
+
+# Toggle Switch
+
+Toggle Switch allows the user to change a setting between two states.
```
Comment (Navaneeth-pk) [inline docs/docs/widgets/toggle-switch.md:7]: `Toggle switch`

File: docs/docs/widgets/toggle-switch.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+---
+sidebar_position: 2
```
Comment (Navaneeth-pk) [inline docs/docs/widgets/toggle-switch.md:2]: Should this be at position `2`?

Comment (arpitnath) [reply docs/docs/widgets/toggle-switch.md:2]: I wanted Toggle to just be below of Radio button. 
or we can serialise all the positions by how they are in the inspector? 
right now we have only 1,2,3 positions

Comment (Navaneeth-pk) [reply docs/docs/widgets/toggle-switch.md:2]: The list should be in alphabetical order. Can you please create another issue for that?

Comment (arpitnath) [reply docs/docs/widgets/toggle-switch.md:2]: yeah sure

**Pull request merged**